schema/plugin.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Table of Contents",
+  "description": "Table of contents settings.",
+  "definitions": {
+    "tocConfig": {
```

Comment:
This does not need to be a nested object. You can elevate the `collapsibleNotebooks` setting. For example, see [here](https://github.com/jupyterlab/jupyterlab-git/blob/master/schema/plugin.json).